packages/web-app/src/views/questionnaires-list.njk

```diff
@@ -0,0 +1,43 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{% set questionnairesListDisplayData = [] %}
+
+{% for questionnaire in sortedQuestionnairesData %}
+  {% if questionnaire.status.text === "Received" %}
+    {% set reference = "<a class='govuk-link' href=\"review-questionnaire-submission/" + questionnaire.id + "\">" + questionnaire.reference + "</a>"%}
+  {% else %}
+    {% set reference = questionnaire.reference%}
+  {% endif %}
+
+  {% set status %}
+    <strong class="govuk-tag govuk-tag--{{ questionnaire.status.colour }}">
+      {{ questionnaire.status.text }}
+    </strong>
+  {% endset %}
+  
+  {% set questionnairesListDisplayData = (questionnairesListDisplayData.push([
+    { html: reference, classes: referenceClass },
```

Comment:
I think we could remove `classes: referenceClass` as `referenceClass` is no longer being set?